### PR TITLE
support SQL case attachments

### PR DESCRIPTION
**Pull request by snopoke**
@czue @benrudolph cc @emord 

:fish: probably easiest

### CODE REVIEW

File: corehq/form_processor/abstract_models.py [hunk 1]
```diff
@@ -130,6 +135,35 @@ def get_actions_for_form(self, xform):
...
+        if attachment_key in self.case_attachments:
+            from dimagi.utils import web
+            from django.core.urlresolvers import reverse
+            return "%s%s" % (web.get_url_base(),
+                             reverse("api_case_attachment", kwargs={
+                                 "domain": self.domain,
+                                 "case_id": self.case_id,
+                                 "attachment_id": attachment_key,
+                             })
+            )
```
Comment (dimagimon) [inline corehq/form_processor/abstract_models.py]: E124 closing bracket does not match visual indentation

**Comment by czue** [comment]
looks good! no blockers, but build fail looks real

```
======================================================================
ERROR: testCreateMultimediaCase (corehq.apps.hqcase.tests.test_object_cache.CaseObjectCacheTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/hqcase/tests/test_object_cache.py", line 65, in testCreateMultimediaCase
    _, _, [case] = self._doCreateCaseWithMultimedia(attachments=attachments)
ValueError: need more than 2 values to unpack
----------------------------------------------------------------------
```

**Comment by emord** [comment]
Merged master, because test wasn't queued. (I think that's how we solve those?)

**Comment by snopoke** [comment]
@emord thanks

**Comment by benrudolph** [comment]
:+1: from me once tests pass

**Comment by snopoke** [comment]
:boot: :construction:

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by snopoke** [comment]
🏄

On 18 February 2016 at 11:06, Cory Zue <notifications@github.com> wrote:

> Merged #10339 <https://github.com/dimagi/commcare-hq/pull/10339>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10339#event-554778784>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa

**Comment by czue** [comment]
:snowboarder:

**Comment by benrudolph** [comment]
:surfer:

**Pull request merged**